### PR TITLE
migrator: add common network errors as retriable errors

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: pkg/migrator/errors.go [hunk 1]
```diff
@@ -43,6 +46,12 @@ type TemporaryError interface {
 	Temporary() bool
 }
 
+// isConnectionRefusedError checks if the error string include "connection refused"
+// TODO: find a "go-way" to detect this error, probably using *os.SyscallError
```
Comment (caesarxuchao) [inline pkg/migrator/errors.go]: Excuse my English, what does "go-way" mean here?

Comment (sttts) [reply pkg/migrator/errors.go]: He means go idiomatic, like with some type check.

**Pull request merged**